src/Compilers/CSharp/Portable/Binder/PatternExplainer.cs

```diff
@@ -403,6 +403,12 @@ string tryHandleListPattern(ref bool unnamedEnumValue)
                     var lengthTemp = new BoundDagTemp(lengthOrCount.Syntax, lengthOrCount.Property.Type, lengthOrCount);
                     var lengthValues = (IValueSet<int>)computeRemainingValues(ValueSetFactory.ForLength, getArray(constraintMap, lengthTemp));
                     int lengthValue = lengthValues.Sample.Int32Value;
+                    if (lengthValue > 20)
```

Comment:
I could not parse past ten elements, doubled it just in case. Let me know if this is too low.

Reply:
When you say could not parse, did you mean visually as a reader?

Reply:
Yup, I don't know how else you could interpret that though 😄